lib/mutant/reporter/cli_compact/printer/mutation_result.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Mutant
+  class Reporter
+    class CliCompact
+      class Printer
+        # Reporter for mutation results
+        class MutationResult < self
+
+          delegate :mutation, :isolation_result
+
+          MAP = {
+            Mutant::Mutation::Evil    => :evil_details,
+            Mutant::Mutation::Neutral => :neutral_details,
+            Mutant::Mutation::Noop    => :noop_details
+          }.freeze
+
+          NEUTRAL_MESSAGE = <<~'MESSAGE'
```

Comment:
I'm not a big fan of this messages being duplicated.

I think we have to share this with the other reporters.